### PR TITLE
Extend Doc Edit Group Details

### PR DESCRIPTION
**Pull request by wattnpapa**
I extende the api doc

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://powerpan-master.c68bfdc4cac4c32c41e1a922e73062ea.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://powerpan-master.c68bfdc4cac4c32c41e1a922e73062ea.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: doc/api/groups.md [hunk 1]
```diff
@@ -34,6 +34,90 @@ Parameters:
...
+ {
+ "id": 1,
+ "name": "Foobar Group",
+ "path": "foo-bar",
+ "owner_id": 18,
+ "description": "An interesting group",
+ "projects" :
+ [
+   {
+    "id": 4,
```
Comment (jvanbaarsen) [inline doc/api/groups.md]: Looks like the indentation here is not correct.

**Comment by jvanbaarsen** [comment]
@PowerPan Can you please fix that one comment I placed, and also squash the commits? Ping me when ready so I can merge :)

**Comment by PowerPan** [comment]
@jvanbaarsen  ok... how can i do it? Must i open a new pull request? or can i edit it directly ? i don't know how ...

**Comment by jvanbaarsen** [comment]
@PowerPan You can read more about this here: http://davidwalsh.name/squash-commits-git

**Comment by jvanbaarsen** [comment]
This merge request has been closed because a request for more information has not been reacted to for more than 2 weeks. If you respond and conform to the merge request guidelines in our [contributing guidelines](https://gitlab.com/gitlab-org/gitlab-ce/blob/master/CONTRIBUTING.md#pull-requests) we will reopen this merge request.

**Pull request closed without merging**